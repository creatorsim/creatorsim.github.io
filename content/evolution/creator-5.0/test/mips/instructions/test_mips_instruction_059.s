
#
# Creator (https://creatorsim.github.io/creator/)
#

    
.text
main:
    li.d $FP0, 256.0
  	li.d $FP2, 3481.0
    li.d $FP4, 5476.0
    
    li.s $f6, 9801.0
    li.s $f7, 8281.0
    li.s $f8, 9216.0
    
    add.d $FP10, $FP0, $FP0
    add.d $FP12, $FP2, $FP2
    add.d $FP14, $FP4, $FP4
    
    add.s $f16, $f6, $f6
    add.s $f17, $f7, $f7
    add.s $f18, $f8, $f8
    
    li $v0, 10
    syscall

