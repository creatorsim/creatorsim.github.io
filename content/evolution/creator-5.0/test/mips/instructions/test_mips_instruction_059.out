
cr[PC]:0x8c; ir[1,at]:0x46100000; ir[2,v0]:0xa; sfpr[f0]:0x40700000; sfpr[f2]:0x40AB3200; sfpr[f4]:0x40B56400; sfpr[f6]:0x46192400; sfpr[f7]:0x46016400; sfpr[f8]:0x46100000; sfpr[f10]:0x40800000; sfpr[f12]:0x40BB3200; sfpr[f14]:0x40C56400; sfpr[f16]:0x46992400; sfpr[f17]:0x46816400; sfpr[f18]:0x46900000; dfpr[FP0]:0x4070000000000000; dfpr[FP2]:0x40AB320000000000; dfpr[FP4]:0x40B5640000000000; dfpr[FP6]:0x4619240046016400; dfpr[FP8]:0x4610000000000000; dfpr[FP10]:0x4080000000000000; dfpr[FP12]:0x40BB320000000000; dfpr[FP14]:0x40C5640000000000; dfpr[FP16]:0x4699240046816400; dfpr[FP18]:0x4690000000000000; keyboard[0x0]:''; display[0x0]:''; 

